artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/ssl/SSLSupport.java

```diff
@@ -351,6 +353,14 @@ private static KeyStore loadKeystore(final String keystoreProvider,
       return ks;
    }
 
+   private static void checkPemProviderLoaded(String keystoreType) {
+      if (keystoreType != null && keystoreType.startsWith("PEM")) {
+         if (Security.getProvider("PEM") == null) {
+            Security.insertProviderAt(new PemKeyStoreProvider(), Integer.parseInt(System.getProperty("artemis.pemProvider.insertAt", "0"), 10));
+         }
+      }
+   }
```

Comment:
it inserts at the start, at the moment I am not aware of any other PEM keystore type provider, but order may be important to some in the future, hence the possibility to configure if that ever arises.

Reply:
it is not totally intuitive, but order (or someones need for it) is the important bit. corrected. thanks!

Reply:
So, original question - do we need to bother with specifying the radix 10 when it is 10 by default? Would be far more readable without it.

Reply:
so you don't have to know the default. for me, i like to see it provided rather than have to peek under the hood to see. Don't have a strong opinion on this one.

Reply:
You think people expect non-10 numbers to be specified? It confusingly makes it look like you are specifying a position for the provider. There is no reason to specify this here when an API exists entirely so you dont have to.

Reply:
ok, removed! thanks